packages/main/src/plugin/kubernetes/kubernetes-context-state.ts

```diff
@@ -52,21 +52,20 @@ import {
 } from '@kubernetes/client-node';
```

Comment:
I noticed we have this file named `kubernetes-context-state` and another in the same folder `context-state`, is this expected ?

Reply:
I would like to work on renaming files/classes globally in the latest PR (https://github.com/containers/podman-desktop/issues/8775#issuecomment-2358182299)

Reply:
👍